students/terrance_jones/lesson9/mailroom9.py

```diff
@@ -0,0 +1,177 @@
+"""
+Mailroom9.py
+Written by Terrance Jones
+Assignment for lesson 9 UW Selfpaced Online course
+"""
+
+class Donor:
+    """Creates a donor with name and and empty dontation list """
+    def __init__(self, name, donations=None):
+        self._name = name
+        if donations is None:
+            self._donations = []
+        else:
+            self._donations = [donations]
+       
+    @property
+    def name(self):
+        """Returns donor name"""
+        return self._name
+
+    @property
+    def donations(self):
+        """Returns donations list for donor"""
+        return self._donations
+
+    def append_donations(self, value):
+        self.donations.append(value)
+
+    @classmethod
+    def thank_you_letter(self, donor_name, donation_amount):
+        dic = {'name': donor_name , 'amount_donated': donation_amount}
+        return("Dear {name}, \n Thank you for your generous donation of ${amount_donated:,.2f}. \nSincerely, Mailroom.".format(**dic))
+
+
+class DonorCollection:
+
+    global donor_list
+    donor_list = {'Carlos Santos':[25,50,100], 'Esperanza Gomez': [10,20,30], 'Paul Jackson':[5,10,15], 'Karl Black':[100,200,300], 'Charles Exx': [15,30,45]}
+
+    @classmethod
+    def donor_exists(self, donor):
+        return donor in donor_list
```

Comment:
in classmethods, you use `cls` var per convention instead of `self` - but these don't need to be classmethods as you don't need access to the class - these can be staticmethods BUT if all your methods are static methods then you're not utilizing OO (as per above comment)